src/test/java/lotto/domain/LottoTest.java

```diff
@@ -0,0 +1,57 @@
+package lotto.domain;
+
+import static org.assertj.core.api.Assertions.*;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+import org.junit.jupiter.api.DisplayNameGeneration;
+import org.junit.jupiter.api.DisplayNameGenerator;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+@DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
+class LottoTest {
+
+    @Test
+    void 생성자_정상입력_생성성공() {
+        assertThatCode(() -> createLotto(1, 2, 3, 4, 5, 6)).doesNotThrowAnyException();
```

Comment:
```suggestion
        assertThatCode(() -> new Lotto(1, 2, 3, 4, 5, 6)).doesNotThrowAnyException();
```
Lotto를 생성하는데 어려움이 있다보니 createLotto() 메서드가 등장함
위와 같이 Lotto의 부 생성자로 int 값을 가변인자로 구현하면 위와 같이 구현할 수 있지 않을까?

Reply:
테스트 편의를 위한 부생성자 추가를 의식하고 있었는데, 습관적으로 헬퍼 메서드를 먼저 만들었네요. 
앞으로 테스트 객체 생성이 반복될 때 부생성자 도입을 먼저 검토하겠습니다!